reconcile/dashdotdb_dvo.py

```diff
@@ -31,26 +32,26 @@ class PrometheusInfo:
 
 
 class DashdotdbDVO(DashdotdbBase):
-    def __init__(self, dry_run, thread_pool_size):
+    def __init__(self, dry_run: bool, thread_pool_size: int):
```

Comment:
I think you can add a return type for this if you add

```
from __future__ import annotations
```

Reply:
Not sure I understand. The return type of `__init__` must always be `None` right? Even importing 

```
from __future__ import annotations
```

doesnt change that.

Reply:
nah, forget about me. My head works intermittently these days 😅